mitielib/include/mitie.h

```diff
@@ -132,6 +132,23 @@ extern "C"
             - If the object can't be created then this function returns NULL.
     !*/
 
+    MITIE_EXPORT mitie_named_entity_extractor* mitie_load_named_entity_extractor_pure_model (
+        const char* filename,
+        const char* fe_filename
+    );
+    /*!
+        requires
+            - filename == a valid pointer to a NULL terminated C string
+            - fe_filename == a valid pointer to a NULL terminated C string
+        ensures
+            - Reads a saved MITIE named entity extractor from disk and returns a pointer to
+              the entity extractor object.
+            - filname must point to a serialized pure_model object, 
```

Comment:
What is a pure_model?  The references to classes should use the full class name so it's clear what class is being referenced.